packages/client/src/raffles/__tests__/getRaffle.test.ts

```diff
@@ -0,0 +1,30 @@
+import { mockRaffleResponse, raffleId } from 'tests/__fixtures__/raffles';
+import client from '../../helpers/client';
+import fixtures from '../__fixtures__/getRaffle.textures';
+import getRaffle from '../getRaffle';
```

Comment:
Suggestion: prefer importing from the index file so we can guarantee coverage of those files. Same for other tests.

Reply:
Closing for now, because changing the import is causing tests to break.